Textile/Style/Typography/FontDescriptor.swift

```diff
@@ -55,6 +59,16 @@ public struct FontDescriptor {
     func with(slant: Slant) -> FontDescriptor {
         FontDescriptor(family: family, size: size, weight: weight, slant: slant)
     }
+    
+    func adjustingSize(by adjustment: Int) -> FontDescriptor {
```

Comment:
What do you think of extending this one step further and adding `adjusting(by: Int)` to `FontSize`? That might make this a bit nicer, like so:

```swift
return FontDescriptor(family: family, size: size?.adjusting(by: adjustment), weight: weight, slant: slant)
```

Reply:
Yep, that ends up cleaner! Something else that's popping up in my head: naming this function `adjusted(by:)` vs `adjusting(by:)` (-ed vs -ing). What are your thoughts?

Reply:
yeah, I like `adjusted` too